controller/src/beerocks/master/son_master_thread.cpp

```diff
@@ -375,6 +375,26 @@ bool master_thread::handle_cmdu_1905_autoconfiguration_search(Socket *sd,
     return son_actions::send_cmdu_to_agent(sd, cmdu_tx);
 }
 
+/**
+ * @brief Check if the operating class of @a radio_basic_caps match the operating classes of
+ *        @a bss_info_conf
+ *
+ * @param radio_basic_caps The AP Radio Basic Capabilities TLV of the radio
+ * @param bss_info_conf The BSS Info we try to configure
+ * @return true if one of the operating classes overlaps, false if they are disjoint
+ */
+static bool has_matching_operating_class(wfa_map::tlvApRadioBasicCapabilities &radio_basic_caps,
+                                         const db::bss_info_conf_t &bss_info_conf)
+{
```

Comment:
In the past, there was a decision no to add static/helper functions to the son_master_thread since it was big enough, and instead, put them on son_actions.
I don't say it is right, but it is a convention.